charts/prometheus-smartctl-exporter/templates/prometheusrule.yaml

```
@@ -13,6 +13,9 @@ metadata:
 {{- end }}
 spec:
   groups:
-  - name: smartctl-exporter.rules
-{{ .Files.Get "rules/rules.txt" | indent 4 }}
+    - name: {{ template "prometheus-smartctl-exporter.name" $ }}
+      rules:
+      {{- range .Values.prometheusRules.rules }}
+        - {{ toYaml . | nindent 10 | trim }}
```

how about something like (untested):
```suggestion
        {{- if .enabled }}
        - {{ toYaml (unset (deepCopy .) "enabled" ) | nindent 10 | trim }}
        {{- end }}
```

---

huh, what a nice trick! it seems to template as expected